lib/powerpoint.js

```diff
@@ -93,21 +110,25 @@ class Powerpoint extends EventEmitter2 {
    * @private
    */
   _convertWithGhostScript (outputDir, pdfFiles, options, callback) {
-    var start = this.nowInMillis()
+    const start = this.nowInMillis()
 
-    const gsExecutable = os.platform() === 'win32' ? GS_PATH || 'gswin32c.exe' : 'gs'
     const imgDir = path.resolve(outputDir, 'img')
     const co = this._getConvertOptions(options)
-    const gsCmdRoot = `"${gsExecutable}" -q -dQUIET -dSAFER -sDEVICE=pngalpha -dMaxBitmap=500000000 -r${co.density} -dUseArtBox`
+    // -dBackgroundColor=16#ffffff
```

Comment:
I'm confused.  Are these options that you decided not to use that you're just documenting here?